### PR TITLE
JBTIS-633 - Add tests for global elements in Camel Editor

### PR DESCRIPTION
**Pull request by djelinek**

### CODE REVIEW

**Comment by vpakan** [comment]
Can one of the admins verify this patch?

**Comment by djelinek** [comment]
Do not merge this because the project wizard has changed

**Comment by djelinek** [comment]
test this

File: plugins/org.jboss.tools.fuse.reddeer/src/org/jboss/tools/fuse/reddeer/editor/CamelEndpointDialog.java [hunk 1]
```diff
@@ -0,0 +1,151 @@
...
+	}
+	
+	public PushButton getCancel() {
+		return new PushButton("Cancel");
+	}
+	
+	/**
+	 * Click on the Finish button in Camel endpoint dialog<br/>
+	 */
+	public void Finish() {
```
Comment (apodhrad) [inline plugins/org.jboss.tools.fuse.reddeer/src/org/jboss/tools/fuse/reddeer/editor/CamelEndpointDialog.java]: Please do not start with a capital in a method name

**Comment by tsedmik** [comment]
You can adapt your tests now. Manipulation with new wizard _New Fuse Integration Project_ is in master branch.

File: plugins/org.jboss.tools.fuse.reddeer/src/org/jboss/tools/fuse/reddeer/editor/CamelEndpointDialog.java [hunk 2]
```diff
@@ -0,0 +1,151 @@
...
+import org.jboss.reddeer.swt.impl.text.LabeledText;
+import org.jboss.reddeer.swt.impl.tree.DefaultTree;
+import org.jboss.reddeer.swt.impl.tree.DefaultTreeItem;
+
+/**
+ * Manipulates with dialog (Wizard) for adding Global Camel Endpoints
+ * 
+ * @author djelinek
+ */
+public class CamelEndpointDialog {
```
Comment (apodhrad) [inline plugins/org.jboss.tools.fuse.reddeer/src/org/jboss/tools/fuse/reddeer/editor/CamelEndpointDialog.java]: Please use a parent class WizardDialod. Then, you needn't to implement methods such as cancel() or finish().

**Comment by apodhrad** [comment]
I added some changes since I need this PR to continue with my task ;-)

pushed to master

**Pull request closed without merging**